Storage/History.swift

```diff
@@ -40,6 +37,11 @@ public protocol BrowserHistory {
     func getPinnedTopSites() -> Deferred<Maybe<Cursor<Site>>>
 }
 
+/** An interface for fast repeated frecency queries. */
+public protocol OptimizedFrecency {
```

Comment:
This should probably be called something like `FrecentHistory`.

Reply:
A better name yeah. But I don't understand the distinction between `Frecent` and `Frecency`. Can I just use the latter? (I wouldn't think to search the code for the word Frecent for instance)